fixtures/fixture.only.spec.yaml

```diff
@@ -0,0 +1,26 @@
+apiCalls:
+    swagger:
+      - name: Login - success
+        only: true
+        call: post_users_login
+        header:
+          Content-Type: application/json
+        data:
+          email: !faker "internet.email"
+          password: !faker ["internet.password", "file"]
+        expect:
+          status: 200
+          json:
+            - user.status: ENABLED
+        save:
+          userResponse: json
+
+
+tests:
```

Comment:
This "tests" section would ideally never be written since the expect in the call should be all we ever need. However, there are many scenarios where this is the only way to test the outcome of a sequence of tests (ie. make sure that the new ID returned from a create user call is the same as the href returned from a search). However, this test section comes after all the calls are made. A pattern that showed up in the very first files written using the early framework was that these tests were hundreds of lines below the actual calls. It was difficult to keep track of what was being saved and what was later tested.

So, in the documentation for this framework, we should strongly discourage this pattern. The tests section is only to be used if the expect section of the calls can't implement the assertion desired.

Nothing to worry for this PR, except to maybe remove this test section from examples. In this case (I have not looked yet), you are probably asserting that these tests don't run, so you need them.

Reply:
I agree 100%. @testbetter this is something to you to be aware of as well.